.github/workflows/ci.yml

```diff
@@ -0,0 +1,212 @@
+name: Node CI Suite
+
+on:
+  push
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node-version: [14.x, 16.x]
```

Comment:
Originally, this script also used a windows-latest runner in addition to MacOS and Ubuntu; however we were getting errors in our unit tests. Need to determine what's going on and add Windows back in. The seven failing unit tests are all producing the following:
    TypeError: Cannot read properties of undefined (reading 'stdin') [some fail at 'stdout']

I have looked at the code shown in the stack trace, but don't see anything that should produce this error. Details can be viewed in the [Actions run results](https://github.com/heroku/cli/actions/runs/3199433397/jobs/5225616053).